src/Microsoft.Sbom.Api/Executors/LicenseInformationFetcher.cs

```diff
@@ -14,8 +14,9 @@
 
 namespace Microsoft.Sbom.Api.Executors;
 
-public class LicenseInformationFetcher : ILicenseInformationFetcher
+public class LicenseInformationFetcher : ILicenseInformationFetcher2
 {
+    public const int DefaultTimeoutInSeconds = 30;
```

Comment:
This might be better in the `Microsoft.Sbom.Common.Constants` class. It puts it with other constants and avoids coupling to concrete execution classes